packages/integrations/mdx/README.md

```diff
@@ -222,6 +223,46 @@ const { frontmatter } = Astro.props;
 </html>
 ```
 
+You can also add type safety using [the `Props` type](/en/guides/typescript/#component-props) with the `MDXLayoutProps` helper.
+
+_**Note:** `MDXLayoutProps` is the same as the `MarkdownLayoutProps` utility type, with `rawContent()` and `compiledContent()` removed (since these are not available for `.mdx` files). If you understand this difference, feel free to use `MarkdownLayoutProps` instead when sharing a layout across `.md` and `.mdx` files._
```

Comment:
```suggestion
:::note
 `MDXLayoutProps` is the same as the `MarkdownLayoutProps` utility type, with `rawContent()` and `compiledContent()` removed (since these are not available for `.mdx` files). This allows you to use `MarkdownLayoutProps` for type safety when sharing a layout across `.md` and `.mdx` files.
 :::
```
I didn't *love* the "if you understand this difference." Can we maybe get away with taking that out, and trust that it's implied that if you're not using those two things in your layout, you're good? Or, is there another "caveat/warning" we can use that more explicitly tacks on at the end "as long as you're not using these things?"

Reply:
Fair enough! I was a bit confused reading that redraft of the last sentence (okay I'm _allowed_ to, but what does Astro recommend?), so I kept the wording mostly the same with "if you understand..." removed. Lmk if that works!